deploy-jenkins/templates/jenkins-persistent-template.j2

```diff
@@ -262,7 +262,7 @@
       "name": "ENABLE_OAUTH",
       "displayName": "Enable OAuth in Jenkins",
       "description": "Whether to enable OAuth OpenShift integration. If false, the static account 'admin' will be initialized with the password 'password'.",
-      "value": "true"
+      "value": "{{ enable_oauth }}"
```

Comment:
we can also add {{ enable_oauth | default('false') }} in case there is no default value provided in the defaults file for some reason. i.e. user may have removed it.